api/v1/flow/flow.proto

```diff
@@ -335,7 +335,7 @@ enum DropReason {
     // non-drop reasons
     DROP_REASON_UNKNOWN = 0;
     // drop reasons
-    INVALID_SOURCE_MAC = 130;
+    HOST_NOT_READY = 130;
```

Comment:
Don't reuse a field number as it messes up deserialization. A client with an older definition of the API would report `INVALID_SOURCE_MAC` and not `HOST_NOT_READY`.
Best practice here if this field is unused is to  mark it as deprecated like below and pick the next available field number for `HOST_NOT_READY`.
```suggestion
    // Deprecated: this drop reason is not emitted since Cilium version x.y.z
    HOST_NOT_READY = 130 [deprecated = true];
```